apps/docs/next.config.mjs

```diff
@@ -111,6 +111,27 @@ const nextConfig = {
             value: 'DENY',
           },
         ],
+        has: [
+          {
+            type: 'host',
+            value: 'supabase.com',
+          },
+        ],
+      },
+      {
+        source: '/:path*',
+        headers: [
+          {
```

Comment:
Do we want to preserve the other headers here? (sts and x-frame-options)

Reply:
Up to you. No strong opinions and I thought things already merged 😅

Reply:
haha 😄 cool I'll add them in and merge